src/features/CourtCaseFilters/CourtCaseFilter.stories.tsx

```diff
@@ -43,6 +44,7 @@ export const WhenThereAreFiltersApplied: ComponentStory<typeof CourtCaseFilter>
     defendantName={"Wilma Flintstone"}
     ptiurn={"Case0000"}
     reasonSearch={"HO200212"}
+    myCaseState={"View cases allocated to me"}
```

Comment:
suggestion: it seems like this filter parameter only has two states, on or off, and using a boolean value could simplify this, as well as keeping the url a bit neater when this is set